slsDetectorSoftware/src/Module.cpp

```diff
@@ -3404,57 +3406,54 @@ sls_detector_module Module::readSettingsFile(const std::string &fname,
     return myMod;
 }
 
+std::string Module::calculateChecksum(char *buffer, ssize_t bytes) {
```

Comment:
I think we can put this in it's own class, then it's easier to test since it' doesn't depend on the Module. Same folder as md5.c?